Library/Formula/zsh.rb

```diff
@@ -17,8 +18,11 @@ class Zsh < Formula
 
   depends_on "gdbm"
   depends_on "pcre"
+  depends_on "autoconf" => :build if build.head?
```

Comment:
Okay, so where should I put it? Before `bottle do` or after all other `depends_on`?